staff/jose-rodriguez/precourse/bingo/index.js

```diff
@@ -0,0 +1,156 @@
+var LISTA_NUMS = [];
+  for (var i = 1; i<100; i++) {
+    LISTA_NUMS.push(i);
+  }
+
+function Number(){
+    var index = Math.floor(Math.random()*LISTA_NUMS.length)
```

Comment:
this kind of variable that is not going to mutate is better to declare it as const

Reply:
thanks for the comment. I didn't use JS ES6 in the precourse